content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -35,16 +36,61 @@ profiles that give only the necessary privileges to your container processes.
 
 ## {{% heading "prerequisites" %}}
 
+{{< version-check >}}
+
 In order to complete all steps in this tutorial, you must install
 [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and
 [kubectl](/docs/tasks/tools/). This tutorial will show examples
-with both alpha (pre-v1.19) and generally available seccomp functionality, so
+with both alpha (pre-v1.22) and generally available seccomp functionality, so
 make sure that your cluster is [configured
 correctly](https://kind.sigs.k8s.io/docs/user/quick-start/#setting-kubernetes-version)
 for the version you are using.
 
 <!-- steps -->
 
+## Enable the `RuntimeDefault` seccomp profile for all workloads
+
+{{< feature-state state="alpha" >}}
+
+A new kubelet alpha [feature gate](/docs/reference/command-line-tools-reference/feature-gates)
+`SeccompDefault` has been introduced with the release of Kubernetes v1.22.0.
+
+If enabled, the kubelet will always use the `RuntimeDefault` seccomp profile of
+the corresponding container runtime instead of running the workload in
+`Unconfined` (seccomp disabled) mode. Those profiles aim to provide a strong set
+of security defaults while preserving the functionality of the workload. It is
+possible that the default profiles differ between container runtimes and their
+release versions, for example when comparing those from
+[CRI-O](https://github.com/cri-o/cri-o/blob/b99366680904420974463572f6f0b92166437b0e/vendor/github.com/containers/common/pkg/seccomp/default_linux.go#L45)
+and
+[containerd](https://github.com/containerd/containerd/blob/261c107ffc4ff681bc73988f64e3f60c32233b37/contrib/seccomp/seccomp_default.go#L51).
+
+Some workloads may require a lower set of syscall restrictions. This means that
+they can fail during runtime even with the `RuntimeDefault` profile. To
+mitigate such a failure, either:
+
+- Run the workload explicitly as `Unconfined`.
+- Disable the `SeccompDefault` feature for the nodes.
+- Create a custom seccomp profile for the workload.
+
+It is recommended to enable the feature on a per-node basis and test the
+execution of workloads in conjunction with the `RuntimeDefault` profile before
+deploying them into production.
+
+Since the feature is in alpha state it is disabled per default. To enable it,
+pass the flag `--feature-gates=SeccompDefault=true` to the `kubelet` CLI or
+enable it via the [kubelet configuration
+file](/docs/tasks/administer-cluster/kubelet-config-file). To enable the feature
+gate in kind, ensure that it ships the minimum required Kubernetes version as
```

Comment:
Either make `kind` a hyperlink or put it in backticks, so it's not confused with the English word “kind”.

Reply:
Yep, linked it to the default website. :+1: